package.json

```diff
@@ -7,6 +7,7 @@
 		"npm": ">=8.19.0"
 	},
 	"dependencies": {
+		"@the-collab-lab/shopping-list-utils": "^2.2.0",
```

Comment:
Super resourceful find!!!

Reply:
Finding that in the issue info and looking to see if it was installed was the first thing we solved kind of, and it was really useful building this code!